app.json

```diff
@@ -32,6 +32,10 @@
      }
    ],
    "env": {
+     "SITE_URL": {
+       "description": "Access url to the instance e.g. https://<app name>.herokuapp.com without a trailing slash.",
+       "required": true
+     },
```

Comment:
I remember we had issues setting `SITE_URL` during build. Has that changed?

Reply:
Previously, Pontoon was able to detect existing domain automatically after deployment.

I took that approach to simplify deployment as much as it was possible.
However, after a few versions of Django (request.get_host() throws an exception if there's no right SITE_URL defined) the process got harder.
The code was already hard to maintain and required of the magic (lazily loaded django settings).
That's why I decided to remove all unnecessary code in the favor of a small requirement from the user.